frontend/src/components/phones/dashboard/Dashboard.tsx

```diff
@@ -123,52 +123,35 @@ export const PhoneDashboard = () => {
   const phoneControls = (
     <div className={styles["phone-controls-container"]}>
       <div className={styles["phone-controls"]}>
-        <span
-          // This is a <span> rather than a <button> to allow the use of flexbox,
-          // but React Aria makes sure it's correctly recognised as a button
-          // thanks to `elementType` passed to `useToggleButton`:
```

Comment:
🎉  A win for a11y too!

Reply:
That's the positive way to present it; the other is that I initially introduced a loss by incorrectly thinking I couldn't use flexbox in buttons :sweat_smile: